src/sidebar/components/test/hypothesis-app-test.js

```diff
@@ -325,75 +363,113 @@ describe('hypothesisApp', function () {
   });
 
   describe('#logout()', function () {
-    it('calls session.logout()', function () {
-      var ctrl = createController();
-      ctrl.logout();
-      assert.called(fakeSession.logout);
-    });
 
-    it('prompts the user if there are drafts', function () {
-      fakeDrafts.count.returns(1);
-      var ctrl = createController();
+    // Tests shared by both of the contexts below.
+    function doSharedTests() {
```

Comment:
All the tests in this function are pre-existing tests, code unchanged, I've just indented them